Piwik.Tracker.Samples/PiwikTrackerSamples.cs

```diff
@@ -60,22 +59,22 @@ private static void Main(string[] args)
         /// <summary>
         /// Triggers a Goal conversion
         /// </summary>
-        static private void GoalConversion()
+        private static async Task GoalConversionAsync()

```

Comment:
Could you explain why the method does not return "void" since nothing is returned? How come it still compiles?